Makefile

```diff
@@ -1,4 +1,5 @@
 PREFIX ?= /usr/local
+DESTLIB ?= /usr/local/lib64
```

Comment:
this is not correct, if PREFIX is passed but DESTLIB is not, we want DESTLIB to change accordingly (appending /lib64 to PREFIX)